lib/jarvis/commands/merge.rb

```diff
@@ -72,40 +72,42 @@ def execute
     Jarvis::Git.config(git, "user.email", committer_email)
     Jarvis::Git.config(git, "user.name", committer_name)
 
+    if branches.empty?
+      logger.info("No branches given, merging to PR's base branch: #{pull.base.ref}")
+      target_branch = pull.base.ref
+      backport_branches = []
+    else
+      target_branch, *backport_branches = branches
+
+      if pull.base.ref != target_branch
+        raise "First branch in the list doesn't match the PR's target branch. Expected '#{pull.base.ref}', got '#{target_branch}'"
```

Comment:
Same as above, what if the base branch was 7.x and want to land in master & 7.x because the patch applies cleanly? This will definitely ensure things don't land but, should we allow doing it?